airbyte-cdk/python/unit_tests/sources/file_based/scenarios/unstructured_scenarios.py

```diff
@@ -131,7 +131,7 @@
         FileBasedSourceBuilder()
         .set_files(
             {
-                "a.txt": {
```

Comment:
Mind adding two basic test scenarios - one with a txt file and one with json, showing that the unstructured parser handles them and gives the expected output?